### DISCUSSION TITLE
install and running 2 apps

### DISCUSSION THREAD

**Discussion by ridhomanihuruk** [The-Wineskin-Project/WineskinServer#404] [Q&A]
hi!
when i want to run 2 apps what should i do?
can i use the existing installed engine? or create new one?
or i just click create new blank wrapper to create another apps?

**Comment by emendelson** [comment]
Just click Create New Blank Wrapper to create the second one.

**Comment by ridhomanihuruk** [comment]
what engine recommended if i want to build for spec below?

Windows 10/11
i3 Gen++
8GB
1GB Nvidia/amd/intel iris
120GB SSD
DirectX 9.0c
